pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
Since this is an API, which will be the user-facing contract and should be the most stable and consistent, i want to make sure we get it right. Creating a POST action verb called `remove` to delete the REST resource seems to contradict the idea of a truly restful API.